src/actions/set_jwt_validation_policy.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Action, checkType } from "./default_ccf.js";
+import { ccf } from "@microsoft/ccf-app/global";
+import { validationPolicyMapName } from "../authorization/jwt/JwtValidationPolicyMap";
+
+export let action = [
```

Comment:
Are we using these as governance for sandbox?

Reply:
For the sake of documentation, Ronny missed a change showing the removal of the action code from `kms.js` and we confirmed in an offline chat that this PR is fully self contained for it's goal and doesn't rely on future PRs